website/docs/reference/appsmith-framework/context-object.md

```diff
@@ -52,6 +55,95 @@ This object contains all the values associated with the current URL that the use
 }
 ```
 
+#### host
+The host property of the URL interface is a string that includes the hostname, and then, a ```:```, followed by the port of the URL(if the [port](/reference/appsmith-framework/context-object#port) is available).
+
+
+Example:
+```js
+//{{appsmith.URL.host}}
+host:"app.appsmith.com:111"
+```
+
+#### hostName
+The hostname property of the URL is a **string** that holds the **domain of the URL**. 
+
+
+:::info
+hostname is the host name (without the port number).
+:::
+
+Example:
+```js
+//{{appsmith.URL.hostname}}
+hostName:"app.appsmith.com"
+```
+
+
+#### fullPath
+A full-path URL designates an EXACT location (such as a page, app, file, etc.). In addition to the domain and TLD, a full-path URL needs to include the protocol, subdomain (such as "app," "support," etc.), path/destination, and possibly a file extension as well as parameters. A full-path can include:
+
+* Protocol
+* Subdomain
+* Domain Name
+* Top Level Domain (TLD)
+* Path/File
+* Parameters
+
+Example:
+```js
+//{{appsmith.URL.fullPath}}
+fullPath:"https://app.appsmith.com/app/demo-app/page1-6324031aa"
+```
+
+#### pathName
+It is a string made up of a collection of path segments, each of which has the ```/``` character prefixed to it. The empty string will be the value of the pathname property if the URL has no path segments.
+
+Example:
+```js
+//{{appsmith.URL.pathname}}
+pathName:"/app/demo-app/page1-6324031aa"
+```
+
+#### port
+The port property of the URL is a string that contains the port number of the URL.
+
+Example:
+```js
+//{{appsmith.URL.port}}
+port:"localhost:3000/"
+```
+#### protocol
+The protocol property of the URL is a string that represents the protocol scheme of the URL, including the ```:```.
+:::info
+The resource name and the protocol identification are separated from one another by a colon and two forward slashes.
+:::
+
+Example:
+```js
+//{{appsmith.URL.protocol}}
+protocol:"https:"
+```
+
+#### hash
+The ```appsmith.URL.hash``` value is the string after the hashtag (**including #**). The URL fragment identification is followed by a hash symbol (#), which is the hash property of the URL interface.
+
+Example:
+```js
+//{{appsmith.URL.hash}}
+hash:"#n912xhesgo"
+```
+
+#### queryParams
+Query parameters are a predefined set of parameters that are added to the end of a URL.
+They are URL extensions that help define particular content or actions based on the data being delivered. A ```?``` is added at the end of a URL, then a query parameter is added right after it.
```

Comment:
It could continue in the same para. Please remove the line break.

Reply:
Could we rewrite this to make it a bit simpler?

`They are URL extensions that help define particular content or actions based on the data being delivered. A ```?``` is added at the end of a URL, then a query parameter is added right after it.`

Something along the lines:
A URL has all the query parameters appended at the end with a "?" as a separator. 

And then maybe add an example URL as below that you have given.